pkg/kf/testutil/integration.go

```diff
@@ -382,7 +395,7 @@ func KF(t *testing.T, kf KfInvoker) *Kf {
 }
 
 // Push pushes an application.
-func (k *Kf) Push(ctx context.Context, appName string, flags map[string]string) {
+func (k *Kf) Push(ctx context.Context, appName, workingDir string, flags map[string]string) {
```

Comment:
Should `workingDir` be a property on Kf instead? I'm thinking we might want to also use this in buildpacks and a few other places.

Reply:
`workDir` seems redundant to `--path` to me.

Reply:
it was